daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I need to make sure the worker image we are using supports this... stand by ;)

Reply:
ack